mmdeploy/backend/onnxruntime/wrapper.py

```diff
@@ -79,7 +79,7 @@ def forward(self, inputs: Dict[str,
             input_tensor = input_tensor.contiguous()
             if not self.is_cuda_available:
                 input_tensor = input_tensor.cpu()
-            element_type = input_tensor.numpy().dtype
+            element_type = input_tensor.cpu().numpy().dtype
```

Comment:
It might take a long time to move a big tensor if it is not on CPU.
Is there any other way to get the element type? Or should we use a small tensor (slice from input_tensor)?